index.html

```diff
@@ -5,12 +5,260 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <style>
+      body {
+      display: flex;
+      flex-direction: column;
+      justify-content: space-between; 
+      align-items: center;
+      margin: 0;
+      font-family: Roboto, sans-serif;
+      min-height: 100vh;
+    }
+
+    .HeadingBox {
+      background-color: #1F2937;
+      padding: 20px;
+      text-align: center;
+      color: #f2f2f2;
+      width: 100vw;
+      flex-direction: row;
+      align-items: center;
+    }
+
+    .HeaderLogo {
+      color: #F9FAF8;
+      font-weight: bold;
+      font-size: 24px;
+      margin-bottom: 10px;
+      position: absolute;
+      top: 0;
+      left: 0px;
+      padding: 10px; 
+    }
+
+    .WebsiteInfo {
+      font-size: 48px;
+      font: bold;
+      margin-right: 850px;
+      margin-top: 10px;
+      color: #F9FAF8;
+    }
+
+    .WebsiteInfo p {
+      margin: 0px 0;
+    }
+
+    .WebsiteInfo p:last-child {
+      margin-left: -130px;
+    }
+
+    .WebsiteSubtext {
+      margin-right: 800px;
+      font-size: 16px;
+      color: #777777;
+    }
+
+    .WebsiteSubtext p {
+      margin: 0px 0;
+    }
+
+    .WebsiteSubtext p:nth-child(2) {
+      margin-left: -87px;
+    }
+    .WebsiteSubtext p:nth-child(3) {
+      margin-left: -335px;
+    }
+
+    .InfoContainer {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 20px;
+    }
+
+
+  .SomeRandomInformationBox {
+    border-radius: 10px;
+    padding-top: 20px;
+    display: inline-block;
+    border: solid #2341ec 3px;
+    text-align: center;
+    flex: 1;
+    margin-right: 10px;
+    height: 80px;
+    width:  105px;
+    margin-left: 50px;
+    margin-top: 20px;
+  }
+
+
+
+  .InfoContainer {
+    display: flex;
+  }
+
+    .RandomInfo1,
+    .RandomInfo2,
+    .RandomInfo3,
+    .RandomInfo4 {
+      margin-top: 85px;
+    }
+
+    .InspirationalQuoteBox {
+      color: #E5E7EB;
+      width: 1000000px;
```

Comment:
this width is very large.  what this does, is it forces the div that this class is applied to, to be very very wide.  It is so wide, that it is wider than the width of your computer.

When this happens, and you then use `text-align: left;` in the `.Author` style, you cannot see the words in the Author div.  

try removing this width.

Reply:
this will also fix issues with the header links